ensembl/conf/json/macaca_mulatta_vcf.json

```
@@ -1,5 +1,15 @@
 {
   "collections": [
+    {
+      "id": "macaca_mulatta_EVA",
+      "source_name": "EVA",
+      "description": "Variants from EVA",
+      "species": "macaca_mulatta",
+      "assembly": "Mmul_10",
+      "type": "remote",
+      "use_as_source" : 1,
+      "filename_template" : "http://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_mulatta/GCA_003339765.3/GCA_003339765.3_current_ids.vcf.gz"
```

Hey there! Please use HTTPS here. Everything else looks good and is working as expected, thanks!

```suggestion
      "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_mulatta/GCA_003339765.3/GCA_003339765.3_current_ids.vcf.gz"
```